go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -25,11 +25,13 @@ class GoModUpdater
           /go: ([^@\s]+)(?:@[^\s]+)?: .* declares its path as: ([\S]*)/m
         ].freeze
 
-        def initialize(dependencies:, go_mod:, go_sum:, credentials:)
+        def initialize(dependencies:, credentials:, repo_contents_path:,
+                       directory:, tidy:)
```

Comment:
Question: should these (and assumed extensions: `vendor:`, etc) be individual booleans like this?

I think `!@repo_contents_stub` needs to be passed through to avoid things that assume there's a full clone,  🥔 🥔 as to how.

Reply:
For the time being I think that's 👍 would also keep the doors open to make `go_mod_tidy` a configuration option via the config file